dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/validation/DefaultValidationService.java

```diff
@@ -196,6 +197,22 @@ public void setCurrentUserService( CurrentUserService currentUserService )
         return results;
     }
 
+    @Override
+    public ValidationRuleExpressionDetails getValidationRuleExpressionDetails( ValidationAnalysisParams parameters )
+    {
+        ValidationRunContext context = getValidationContext( parameters );
+
+        ValidationRuleExpressionDetails validationRuleExpressionDetails = new ValidationRuleExpressionDetails();
```

Comment:
In cases like this I find it improves readability when the variables get a simpler name like `details` instead of `validationRuleExpressionDetails` so the string becomes more recognizable in the context of the method.

Reply:
Agreed, thanks! Done.